### ISSUE TITLE
workspace-generator does not print the options

### ISSUE THREAD

**Issue by eggp** [nrwl/nx#7939]
## Current Behavior
run command in shell: nx workspace-generator WGNAME help or nx workspace-generator WGNAME --help
result output:
```
Options:
  --help             Show help                                                                                  [boolean]
  --version          Show version number                                                                        [boolean]
  --list-generators  List the available workspace-generators                                                    [boolean]
```

## Expected Behavior
According to schema.json, display the configuration options as for other schematics tools

## Steps to Reproduce
run commands in nx project:
1. nx generate @nrwl/workspace:workspace-generator my-generator
2. nx workspace-generator my-generator help or nx workspace-generator my-generator --help 

### Environment
Used nx packages:
    "@nrwl/cli": "13.2.1",
    "@nrwl/devkit": "13.2.1",
    "@nrwl/node": "13.2.1",
    "@nrwl/tao": "13.2.1",
    "@nrwl/workspace": "13.2.1",

**Comment by tomastrajan** [comment]
Would be great to get fix for this 👍

**Comment by AgentEnder** [comment]
This will be fixed as part of #9116, for generators that are moved over. Docs changes will follow soon after, preferring the use of workspace plugins to workspace generators.
